spec/spec.md

```diff
@@ -19,7 +19,7 @@ redirect_from:
 
 The below are the most stable versions of the lanuguage specification, which are in sync with the Ballerina releases.
 
-> **Note:** The `ChangeLog` section of the specification identifies the changes that have occurred in each version of the specification.
+> **Note:** The `CHANGES SINCE PREVIOUS VERSIONS` section of the specification identifies the changes that have occurred in each version of the specification.
```

Comment:
Should not be all caps and should not use backticks.

Reply:
Updated via 4e658f71f61f723e472e26ea182f19d6b76e27a7